backend/tests/api-e2e/test_api_antisybil.py

```diff
@@ -6,7 +6,41 @@
 
 
 @pytest.mark.api
-def test_antisybil(client: Client, ua_alice: UserAccount):
+def test_holonym(client: Client):
```

Comment:
👏 for API tests but I think we should also cover the service with UTs

Reply:
+1